src/xdocs/google_style.xml

```diff
@@ -727,7 +727,7 @@
                             <td><a name="4.7"/><a href="#4.7"><img src="images/anchor.png" alt=""/></a></td>
                             <td>
                                 <a
-                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s4.7-grouping-parentheses">4.7 Grouping parentheses: recommended</a>
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s4.7-grouping-parentheses">4.7 Grouping parentheses: recommended</a>
```

Comment:
Grouping enums was 4.8.1. This line is for 4.7, grouping parenthesis. Nothing changed here.
@romani 4.8.1 lists no check. Should it say `EmptyLineSeparator`?

@agcuda `between enum values` You mean the token `ENUM_CONSTANT_DEF`? If so [check doesn't accept that token](http://checkstyle.sourceforge.net/config_whitespace.html#EmptyLineSeparator).